ncm-ssh/src/main/pan/components/ssh/schema.pan

```diff
@@ -73,6 +73,7 @@ type ssh_core_options_type = {
     "Compression" ? string with match (SELF, '^(yes|delayed|no)$')
     "GSSAPIAuthentication" ? legacy_binary_affirmation_string
     "GSSAPICleanupCredentials" ? legacy_binary_affirmation_string
+    "GSSAPIKexAlgorithms" ? string
```

Comment:
This should at the very least be `string_trimmed`, but presumably this is a comma separated list like `KexAlgorithms`?

`man` suggests the following values are valid?
```
gss-gex-sha1-
gss-group1-sha1-
gss-group14-sha1-
gss-group14-sha256-
gss-group16-sha512-
gss-nistp256-sha256-
gss-curve25519-sha256-
```

Reply:
makes sense, amended